vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -126,6 +122,8 @@ type internal FSharpProjectOptionsManager
     member this.ClearInfoForSingleFileProject(projectId) =
         singleFileProjectTable.TryRemove(projectId) |> ignore
 
+    member __.CommandLineOptions with get() = commandLineOptions
```

Comment:
You can (and should) remove ``with get()`` to be idiomatic

Reply:
Hmm I need to work on my F# programming skillz :-)